data/com.mesonbuild.install.policy

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC "-//freedesktop//DTD polkit Policy Configuration 1.0//EN"
+"http://www.freedesktop.org/software/polkit/policyconfig-1.dtd">
+<policyconfig>
+
+  <vendor>The Meson Build System</vendor>
+  <vendor_url>https://github.com/mesonbuild/meson</vendor_url>
+
+  <action id="com.mesonbuild.install.run">
+    <description>Install the given project via Meson</description>
+    <message>Authentication is required to install this project</message>
+    <icon_name>audio-x-generic</icon_name>
```

Comment:
Why it has audio icon?

Reply:
@Prototik I...have no clue. I changed it now to `preferences-system`.